pkgs/development/interpreters/babashka/wrapped.nix

```diff
@@ -1,10 +1,12 @@
 { stdenv
+, lib
 , babashka-unwrapped
 , clojure
 , makeWrapper
 , rlwrap
 
 , jdkBabashka ? clojure.jdk
+, enableRlwrap ? false
```

Comment:
Enable looks like it is something that babashka itself offers. I think `withRlwrap` would fit better here.

Reply:
Can we also add an comment here saying what the advantage of enabling this and also the possible issues?

Reply:
@thiagokokada thanks, it makes sense. I updated the PR, let me know if the comments are not clear enough.

Reply:
Looks great, thanks!